libsolidity/analysis/DocStringAnalyser.cpp

```diff
@@ -42,7 +42,14 @@ void copyMissingTags(set<CallableDeclaration const*> const& _baseFunctions, Stru
 	if (_baseFunctions.size() != 1)
 		return;
 
-	auto& sourceDoc = dynamic_cast<StructurallyDocumentedAnnotation const&>((*_baseFunctions.begin())->annotation());
+	auto hasParameter = [](CallableDeclaration const& __declaration, size_t _n)
+	{
+		return __declaration.returnParameterList() &&
+			__declaration.returnParameters().size() > _n;
+	};
+
+	auto& baseFunction = **_baseFunctions.begin();
```

Comment:
Can you move this directly after the `if (_baseFunction.size()` - it makes total sense there. And please also add an explicit type, since this is one of the most important local variables.